distributed/client.py

```diff
@@ -3390,6 +3390,26 @@ def get_task_stream(self, start=None, stop=None, count=None):
         return self.sync(self.scheduler.get_task_stream, start=start,
                          stop=stop, count=count)
 
+    @gen.coroutine
+    def _register_init_func(self, function):
+        response = yield self.scheduler.register_init_func(function=dumps(function))
+        raise gen.Return(response)
+
+    def register_init_func(self, function, asynchronous=None):
+        """
+        Registers and run an initialization for every worker
+
+        This registers a function to be called by every currently connected
+        workers, and that will be also called by any added worker after that.
+        Multiple functions can be added by multiple calls.
+
+        Parameters
+        ----------
+        function: Function to register and run
```

Comment:
This should be `name : type` with the description on the line below. For me, and particularly with functions I find it very useful to include the signature - e.g.
```
        Parameters
        ----------
        function : callable() -> None
            Function to register and run
```
Also note the space around the `:` - at least in spyder it won't render correctly without that.

Reply:
Done.